textproc/epubcheck/Portfile

```diff
@@ -1,43 +1,54 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
+
 PortGroup               java 1.0
 
-name                    epubcheck
-version                 1.2
+PortGroup               github 1.0
+
+github.setup            w3c epubcheck 4.2.2 v
 categories              textproc
 platforms               darwin
-license                 BSD 
+license                 BSD-3-Clause
 maintainers             nomaintainer
 supported_archs         noarch
+java.version            1.7+
+java.fallback           openjdk11
 
-description             ePub Validator to validate ePub documents
+description             EPUB validator
 
-long_description        EpubCheck is a tool to validate IDPF  EPUB files. \
-                        It can detect many types of errors in EPUB. OCF \
-                        container structure, OPF and OPS mark-up, and \
-                        internal reference consistency are checked.
+long_description        EPUBCheck is a tool to validate the \
+                        conformance of EPUB publications against the \
+                        EPUB specifications. EPUBCheck can be run as a \
+                        standalone command-line tool or used as a Java \
+                        library.
 
-homepage                https://code.google.com/p/epubcheck/
-master_sites            googlecode
+checksums               sha256  029ee9cda237a42e0d4c1a2b8771a440a4a6326449db843510b16a3c61045463 \
+                        rmd160  cebf60866877013c2a9170605548d139fd16a232 \
+                        size    11990465
 
-distname                ${name}-${version}
+# fetch phase keywords
+# This Portfile downloads the Java bytecode for EPUBCheck from GitHub,
+# rather than pulling together all the Java bits and pieces necessary
+# to run EPUBCheck
+github.tarball_from     releases
 
+# extract phase keywords
 use_zip                 yes
+extract.post_args       ""
 
-checksums               sha256  2ad436aeaaa341f28e11fdb91256bc04e8b0747d50857f58f5bd97ec6a6cb265 \
-                        rmd160  a086609ba13ae36ec63759cf2f7f474b88c8ccdc \
-                        size    1399350
-
-java.version            1.5+
-java.fallback           openjdk11
-
-extract.mkdir           yes
-
+# configure phase keywords
```

Comment:
Please remove this generic comment.

Reply:
Whoa! Rebased the thing and it blew up.

I'm going to delete this branch and start over.